src/EnergyPlus/SizingManager.cc

```diff
@@ -4235,20 +4333,28 @@ namespace SizingManager {
 	UpdateTermUnitFinalZoneSizing()
 	{
 		// Move data from FinalZoneSizing to TermUnitFinalZoneSizing and apply terminal unit sizing adjustments
-		// Called once to initialize before system sizing, then called again to update after system sizing
+		// Called once to initialize before system sizing
 		// M.J. Witte, July 2017
 
 		for ( int termUnitSizingIndex = 1; termUnitSizingIndex <= DataSizing::NumAirTerminalUnits; ++termUnitSizingIndex ) {
 			auto & thisTUFZSizing( TermUnitFinalZoneSizing( termUnitSizingIndex ) );
-			auto const & thisTUSizing( TermUnitSizing( termUnitSizingIndex ) );
+			auto & thisTUSizing( TermUnitSizing( termUnitSizingIndex ) );
 			int ctrlZoneNum = thisTUSizing.CtrlZoneNum;
 			auto const & thisFZSizing( FinalZoneSizing( ctrlZoneNum ) );
 
 			// Copy everything from FinalZoneSizing to TermUnitFinalZoneSizing
 			thisTUFZSizing = thisFZSizing;
+			thisTUFZSizing.ADUName = thisTUSizing.ADUName;
 
 			if( DataSizing::NumAirTerminalSizingSpec > 0 ) {
 				// Apply DesignSpecification:AirTerminal:Sizing adjustments - default ratios are 1.0
+				Real64 minOAFrac = thisTUSizing.SpecMinOAFrac;
+				// Outdoor air
+				thisTUFZSizing.MinOA = thisFZSizing.MinOA * minOAFrac;
+				thisTUFZSizing.TotalOAFromPeople = thisFZSizing.TotalOAFromPeople * minOAFrac;
+				thisTUFZSizing.TotalOAFromArea = thisFZSizing.TotalOAFromArea * minOAFrac;
```

Comment:
Line 4355 SizingManager. So these are the total zone OA flow per person and zone, correct (from line 1640 of ZoneEquipmentManager)? And you move that data into each terminal unit sizing array which is then used for Std 62.1. Do you know the TU flow rate and the total zone flow rate here? You could use that ratio to allocate OA to each terminal unit. At the very least, you could divide by number of TU's per zone.

Reply:
The user is supposed to manage OA allocation with DesignSpecification:AirTerminal:Sizing objects. I hadn't thought about trying to automatically allocate that. If the user puts more than one air terminal in the same zone without any DSATZ objects, then there will be excess OA. Flow rates are known here, so I suppose something could be done automatically for this case.

Reply:
And just to clarify. TerminalUnitFinalZoneSizing is used for more than Std 62.1 calcs - it's used everywhere for terminal unit sizing, including .MinOA. So, then the question would be if we're going to automatically allocate OA for multiple terminal units without any DesignSpec AT Sizing objects, then do we automatically divide up the supply flow also? If there are no DSATZ objects, then the supply flow rate will be the same for every terminal unit in the same zone. So, that puts us back to dividing by the number of TUs per zone - but *only if nobody in that zone has a DesignSpec AT Sizing object.*